### DISCUSSION TITLE
ATTENZIONE: Svolgimento scrutinio sospensione del giudizio per chi ha urgenza di svolgerlo

### DISCUSSION THREAD

**Discussion by trinko** [iisgiua/giuaschool#736] [Annunci]
Come detto, la prossima versione conterrà le correzioni e le modifiche alla procedura dello scrutinio per gli alunni con sospensione del giudizio, ma non sono sicuro di riuscire a chiudere tutto in poco tempo.

Per cui, chi avesse urgenza di svolgere questo scrutinio può:
- installare gli aggiornamenti successivi all'uscita della **versione 1.5.8**; 
- effettuare un backup completo;
- eseguire la procedura dello scrutinio;
- in caso di problemi, aprire una segnalazione caricando il log degli errori (vedi #576).

**Comment by luigirotondi2009** [comment]
Il backup su altervista come si fa?

**Comment by trinko** [comment]
Per fare il backup dei file devi scaricarli sul tuo computer, usando un programma come FileZilla ([scaricalo da qui](https://filezilla-project.org/download.php?platform=win64) se non ce l’hai). FileZilla ti permette di caricare i file sul tuo spazio Altervista: se non conosci questo programma guarda [questo tutorial](https://www.youtube.com/watch?v=8kOMTK3XOp8).

Per fare il backup del database:
- dal pannello di controllo di Altervista, vai su **Risorse** e nel riquadro **Database** clicca su _Accedi a PHPmYAdmin_;
- a questo punto clicca sulla voce **ESPORTA** che trovi nei pulsanti in alto, nella barra del menu;
- lascia le impostazioni predefinite e clicca su **Esporta**.
Verrà scaricato un file SQL che contiene i comandi per creare la struttura del database e inserire i dati.
Questo file, in caso di necessità, può essere caricato nel database con la funzione IMPORTA di PhpMyAdmin.
